openshift/qontract-manager.yaml

```
@@ -328,6 +328,9 @@ parameters:
   value: ''
 - name: PUSHGATEWAY_SECRET_NAME
   value: push-gateway-auth-url
+- name: SLACK_RECONCILE_ERR_CHANNEL_NAME
+  value: "sd-app-sre-govslack-errors-stage"
```

Not a huge deal, but I suggest this just to clarify that it's not strictly a govslack specific feature.

```suggestion
  value: "sd-app-sre-slack-errors-stage"
```